### ISSUE TITLE
Enquiry about status of this library 

### ISSUE THREAD

**Issue by J0** [OpenSRM/storage#1]
tl;dr: Is this project still actively being maintained/used? We would love to use the PyPI namespace `storage` if it's not being maintained/used but no pressure!

Hey,

I came across this library on PyPI while checking the availability of the PyPI name `storage` for a [python client library](https://github.com/supabase-community/supabase-py) for one of the components of [Supabase](https://supabase.com/) , an open source firebase.  The storage component of Supabase involves an object backed storage(similar to S3) which our library would interact with.

I was hoping to use `storage` as the PyPI package name so that it would be aligned with our other packages `realtime`, `gotrue` , and `supabase`. Our current alternative is to use `storage_py` or a related name but I would love to have `storage` if possible(and I think the community would too). While it seems like a minor detail, we think it would aid users by allowing for a consistent naming scheme.

I noticed that the last commit seems to be in 2016 and there are no active issues so I was wondering if the project is still being actively maintained and if the package `storage` is still being actively used. I realize there may be downloads/activity that I might not be seeing so just thought I'd post an issue to check and take questions(if any)

Let me know!
Joel

**Comment by J0** [comment]
Hi team,

Just letting you know that we've opened an issue on PyPI to request for the namespace

https://github.com/pypa/pypi-support/issues/1591

Do let us know if there are active users and if you'd like to keep the namespace though!
